app/utils/ImplicitDateFormatter.scala

```diff
@@ -44,16 +44,7 @@ trait ImplicitDateFormatter {
 
     val number = if (date.getDayOfMonth < 20) {date.getDayOfMonth} else {date.getDayOfMonth % 10}
 
-    val dateSuffix = number match {
-      case 1 => "st"
-      case 2 => "nd"
-      case 3 => "rd"
-      case _ => "th"
-    }
-
     val outputFormat = messages.lang.code match {
```

Comment:
could you please remove language check and replace with simple if else with dayOfWeek